iroh-bytes/src/baomap.rs

```diff
@@ -140,7 +142,11 @@ pub trait ReadableStore: Map {
     ///
     /// This function should not block to perform io. The knowledge about
     /// existing roots must be present in memory.
-    fn roots(&self) -> Box<dyn Iterator<Item = Hash> + Send + Sync + 'static>;
+    fn tags(&self) -> Box<dyn Iterator<Item = (Bytes, Cid)> + Send + Sync + 'static>;
+
+    /// Temp pins
```

Comment:
can we call them temporary tags? I really dislike the pinning name, it is very overloaded, especially in rust

Reply:
Agree that temp pins does not make sense if the other things are not called pins. But temp tags also is not great, since these are unnamed.

Reply:
renamed it to temp tags for now, but I am not 100% happy with the name

Reply:
I guess the exact name for the thing depends on the outcome of the naming discussion below.

What you get is basically a (hash, format) pair that keeps the corresponding data safe from gc until it gets dropped. It is not just a tag, but also contains the hash and format, so it can be used interchangeably with an actual (hash, format) pair.